vxyowsup/whatsapp.py

```diff
@@ -147,6 +151,21 @@ def onMessage(self, messageProtocolEntity):
 
     @ProtocolEntityCallback("receipt")
     def onReceipt(self, entity):
+        '''receives confirmation of delivery to human'''
+        # shady
+        # getting too many receipts
```

Comment:
Curiously it's no longer an issue, though I don't think I've touched the receipts much. But now it doesn't seem like I'm getting any receipts... :confused: I also thought sending an ack for a receipt is questionable, but I just left it in for now because it was in the tutorial :confused: